app/components/Dashboard/PriceHistoryPanel/PriceHistoryPanel.jsx

```diff
@@ -101,4 +109,49 @@ export default class PriceHistoryPanel extends React.Component<Props> {
     }
     return date.toLocaleString('en-US', { month: 'numeric', day: 'numeric' })
   }
+
+  renderLatestPrice = () => {
+    const { staticPrice } = this.props
+    return (
+      <div className={styles.currentPrice}>
+        {this.formatPrice(staticPrice, formatFiat)}
+      </div>
+    )
+  }
+
+  formatPrice = (
+    price: number,
+    formatter: Function = formatThousands
```

Comment:
generally I think it is better to be more descriptive than Function. But it depends on how many various formatters you might use and their various outputs. This might be ok